examples/kepler-integration/src/components/export-video/export-video-panel-settings.js

```diff
@@ -57,12 +57,58 @@ const InputGrid = styled.div`
   grid-row-gap: ${DEFAULT_ROW_GAP};
 `;
 
+/**
+ * Used to convert durationMs (inherited from ExportVideoPanelContainer) to hh:mm:ss
+ * @param {number} durationMs duration of animation in milliseconds
+ * @returns {string} time in format hh:mm:ss
+ */
+function msConversion(durationMs) {
+  let seconds = parseInt((durationMs / 1000) % 60, 10);
+  let minutes = parseInt((durationMs / (1000 * 60)) % 60, 10);
+  let hours = parseInt((durationMs / (1000 * 60 * 60)) % 24, 10);
+
+  hours = hours < 10 ? `0${hours}` : hours;
+  minutes = minutes < 10 ? `0${minutes}` : minutes;
+  seconds = seconds < 10 ? `0${seconds}` : seconds;
+
+  return `${hours}:${minutes}:${seconds}`;
```

Comment:
Users won't be exporting hour or minute long clips often (or ever). Let's just allow seconds to go up higher then 59. I also think having 1 digit of milliseconds could be good. 
So "130.5 seconds" is what we'd show fir 2 minutes, 10 seconds, 500 milliseconds.

Reply:
I agree on the hour. That's just how it was in the Figma. However, I think minute long clips might be reached once users have the ability to string together different camera angles and positions. I'm not sure about milliseconds though? What's your thinking behind that?

Reply:
This UI serves users exporting a single clip, so I want it be designed for the use case instead of a future one.

Reply:
![image](https://user-images.githubusercontent.com/33266041/98484779-9d3a0a80-21c6-11eb-8f78-4b7ed902fa47.png)
Alright this is what it'll look like in the next push